.github/workflows/main.yml

```diff
@@ -385,12 +385,11 @@ jobs:
           fetch-depth: 0
       - name: Set up StoneDB
         run: |
-          docker pull stoneatom/stonedb:v1.0.3
-          docker run -p 3306:3306 -itd  -e MYSQL_ROOT_PASSWORD='123456'  stoneatom/stonedb:v1.0.3
-          docker ps
-          ContainerID=$(docker ps --filter ancestor=stoneatom/stonedb:v1.0.3 --format "{{.ID}}") 
+          docker run -p 3306:3306 -itd  -e MYSQL_ROOT_PASSWORD='123456'  stoneatom/stonedb
+          ContainerID=$(docker ps --filter ancestor=stoneatom/stonedb --format "{{.ID}}") 
           docker exec $ContainerID bash
-          sleep 180s
+          sleep 60s
+          docker exec $ContainerID sed -i "s/tianmu_insert_delayed=1/tianmu_insert_delayed=0/" /opt/stonedb57/install/my.cnf
```

Comment:
StoneDB team let us set this during testing. Because if not set, there are bugs that they are currently fixing.